dist/bu-blocks-frontend.js

```diff
@@ -570,7 +570,7 @@ const bu_blocks = {};
 	 * Find all Collapsible blocks on a page
 	 */
 	var findAllCollapsibleBlocks = function() {
-		var containers = document.querySelectorAll( '.wp-block-bu-collapsible' );
+		var containers = document.querySelectorAll( '.js-wp-block-bu-collapsible' );
```

Comment:
🤷‍♂️  10up wrote it, I don't intend to refactor it all.

Reply:
Also, generally in the frontend JS I had been trying to avoid `const` and es6 code because: 
1) we don't have babel setup to convert these files
2) unlike the block code itself, I felt like it was important to keep the frontend code for the blocks simpler and more accessible to designers less versed in the latest advanced JS features as they will need to interact with the frontend JS. 

I'm not going to refactor all of the code 10up wrote  here to simplify it more, but I fixed a few things that were breaking Internet Explorer.